x-pack/metricbeat/module/stan/_meta/kibana/7/dashboard/Metricbeat-STAN-overview.json

```diff
@@ -227,14 +180,14 @@
                   {
                     "field": "stan.channels.depth",
                     "id": "61ca57f2-469d-11e7-af02-69e470af7417",
-                    "type": "sum"
+                    "type": "max"
```

Comment:
In hindsight, a max query is probably better here from a monitoring perspective. I chose sum thinking it'd be useful in case we have many hosts with the same channel name.

Reply:
👍